Makefile

```diff
@@ -1,4 +1,4 @@
-REBAR ?= $(shell which rebar 2>/dev/null || which ./rebar)
+REBAR ?= $(shell which rebar3 2>/dev/null || which rebar 2>/dev/null || which ./rebar)
```

Comment:
Can we just remove support for the old `rebar` entirely, please?

Reply:
Or just not change the `Makefile` and `README.md` files at all since this is a forked public library.

Reply:
Sure, will remove the old `rebar`.

Reply:
Done!